libraries/AP_BattMonitor/AP_BattMonitor_UAVCAN.h

```diff
@@ -29,12 +29,17 @@ class AP_BattMonitor_UAVCAN : public AP_BattMonitor_Backend
     }
 
     static void subscribe_msgs(AP_UAVCAN* ap_uavcan);
-    static AP_BattMonitor_UAVCAN* get_uavcan_backend(AP_UAVCAN* ap_uavcan, uint8_t node_id);
+    static AP_BattMonitor_UAVCAN* get_uavcan_backend(AP_UAVCAN* ap_uavcan, uint8_t node_id, uint8_t battery_id);
     static void handle_battery_info_trampoline(AP_UAVCAN* ap_uavcan, uint8_t node_id, const BattInfoCb &cb);
 
 private:
     void handle_battery_info(const BattInfoCb &cb);
 
+    static bool this_is_the_battery_youre_looking_for(uint8_t instance, uint8_t battery_id) {
```

Comment:
cute fn name :-)
I'd prefer match_battery_id() I think, sorry to be a party pooper

Reply:
reluctantly changed to match_battery_id() even though I had THE PERFECT fn name...